### PR TITLE
fix [object Object] ids on trigger and menu

### PR DESCRIPTION
**Pull request by diaasaur**
fixes: #703 

noticed this error in browser console and it looks like an issue of value mismatch for ids (trigger + menu)
Screenshot 2023-11-08 at 1 24 48 AM

### CODE REVIEW

File: src/lib/builders/context-menu/create.ts [hunk 1]
```diff
@@ -138,8 +138,8 @@ export function createContextMenu(props?: CreateContextMenuProps) {
 				style: styleToString({
 					display: $isVisible ? undefined : 'none',
 				}),
-				id: ids.menu,
-				'aria-labelledby': ids.trigger,
+				id: get(ids.menu),
```
Comment (TGlide) [inline src/lib/builders/context-menu/create.ts:141]: These should be reactive, meaning you should reference ids.menu in `stores`, then use it in `returned`.

**Review by TGlide — CHANGES_REQUESTED** [review]
Thank you! Just one thing that needs to be fixed.

**Comment by huntabyte** [comment]
Just now seeing this but I already took care of this as well as the submenus and some other issues + tests in #707

**Comment by diaasaur** [comment]
> Just now saw this but I already took care of this as well as the submenus and some other issues (like the screenshot of the DOM Exception) + tests in #707

no worries! I will close mine since your PR seems to take care of tests as well :D

**Pull request closed without merging**